### DISCUSSION TITLE
Just updated cosmos, can't open my previous vs project

### DISCUSSION THREAD

**Discussion by EnZon3** [CosmosOS/Cosmos#2144] [Q&A]
![image](https://user-images.githubusercontent.com/37979038/156180578-badd5026-573b-44fd-855d-c0a9fbd97e41.png)
This is all I see when I try to open the project file.

**Comment by EnZon3** [comment]
I was using .NET 2.0

**Comment by quajak** [comment]
It looks like you did not open any solution

**Reply by EnZon3** [reply]
I did though...

**Reply by quajak** [reply]
If you opened correctly, there should be something to see in the Solution Explorer unless you are getting any error when you are opening it?

**Reply by EnZon3** [reply]
No, I got nothing, I opened it like normal and I already tried the solution, VS stopped responding and then displayed nothing.

**Comment by EnZon3** [comment]
Ok, I found a way to open it

I just opened it directly from the sln file

**Reply by EnZon3** [reply]
But the dependencies won't load for some reason.

**Reply by EnZon3** [reply]
I'll try to fix this myself.
